features/proposed/FxA-70-kpi-dashboards/README.md

```diff
@@ -0,0 +1,567 @@
+
+2016 KPI Dashboards
+===================
+
+https://mozilla.aha.io/features/FXA-70
+
+## Stories
+
+As a product stakeholder for Firefox Accounts
+I want to quickly see the key metrics we care about for the year
+and how they're changing over time.
+
+## Overview
+
+The top-line objective for FxA in 2016 is to
+*seamlessly connect users to a compelling Firefox experience
+on all their devices*.
+To support this we will build a KPI dashboard
+that focusses on three key metrics:
+
+* Engagement Ratio,
+  as a measure of whether
+  they find the experience compelling.
+* Multi-Device Usage,
+  as a measure of how well we are
+  being present on all the user's devices.
+* Time taken for device connection,
+  as a measure of how seamlessly
+  we are making the connection.
+
+By default,
+each of these measures will be presented
+as an aggregation over all users.
+Linked detail dashboards will display them
+broken out by key user segmentation characteristics
+such as browser version,
+os platform,
+and whether they are a multi-device user.
+
+### Engagement Ratio
+
+This is the ratio of daily active users
+as a percentage of monthly active users,
+and is a measure of whether we have
+a compelling Firefox Experience.
+It does not meaningfully represent ongoing engagement
+with Firefox Accounts itself,
+since we're largely a background service
+once the user has connected their devices.
+
+To reflect this dependence on broader Firefox engagement levels,
+our target for 2016 will be twofold:
+
+* Increase engagement ratio by 5 percentage points over 2015 baseline.
+* Achieve at least 5 percentage points *higher* engagement ratio
+  than the overall ratio for each of Firefox desktop, Android and iOS.
+
+TODO: establish baselines
+
+TODO: sanity-check reasonableness of goals
+
+The top-level KPI dashboard will show a graph of
+engagement ratio, calculated daily,
+for Firefox Accounts
+alongside each of Firefox desktop, Android and iOS.
+It will also show the engagement ratio
+specifically for multi-device users.
+It will look something like this:
+
+---
+![](engagement-ratio-graph.png)
+---
+
+A linked detail dashboard can present
+engagement ratio alongside the underlying DAU and MAU,
+broken down by platform
+and by single/multi-device users.
+This will be designed and implemented
+as follow-up work.
+
+### Multi-device usage
```

Comment:
I know this PR is already merged, but I had a couple questions. What is considered a device?

* Access Firefox accounts through Firefox browser (Phone and Desktop)?
* Access Firefox accounts through Native browser (Phone)?
* Does user have to sync settings (Phone and Desktop) to consider new device?